crates/data_components/src/object/text.rs

```diff
@@ -81,24 +81,20 @@ impl ObjectStoreTextTable {
         ])
     }
 
-    fn to_record_batch(
-        meta_list: &[ObjectMeta],
-        raw: &[Bytes],
-        formatter: Option<&Arc<dyn DocumentParser>>,
-    ) -> Result<RecordBatch, ArrowError> {
-        if meta_list.len() != raw.len() {
-            return Err(ArrowError::ParseError("Length mismatch".to_string()));
-        }
-
-        let schema = Self::table_schema();
-
-        let location_array: ArrayRef = Arc::new(StringArray::from(
+    fn get_location_value(meta_list: &[ObjectMeta]) -> ArrayRef {
```

Comment:
@Advayp let's make sure we have a snapshot test before and after, so we can see the explain plan is correct for this.

Reply:
Yes, let's add a test for this.